test/resources/cartridge_collection.json

```diff
@@ -0,0 +1,14 @@
+{
+	"name": "Test_Cartridge_Collection",
+	"cartridges": [{
+		"folder": {
+			"name": "Skeleton_Cartridge",
+			"display_name": "Skeleton Cartridge",
+			"description": "This is the skeleton cartridge"
+		},
+		"cartridge": {
+			"url": "https://github.com/RobertNorthard/adop-cartridge-skeleton.git",
```

Comment:
Thanks @anton-kasperovich updated to use skeleton cartridge under the Accenture namespace 👍